src/views/Nft/market/components/BuySellModals/SellModal/index.tsx

```diff
@@ -138,6 +142,37 @@ const SellModal: React.FC<SellModalProps> = ({ variant, nftToSell, onDismiss })
     setStage(SellingStage.TRANSFER)
   }
 
+  const dispatchSuccessAction = () => {
+    // Remove from sale
+    if (stage === SellingStage.CONFIRM_REMOVE_FROM_MARKET) {
```

Comment:
I think this can be switch case (to avoid a bit ugly empty returns). But that's just my taste, feel free to leave it like this